.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-lib
+# lib
```

Comment:
`lib` needs to be included in the gitignore file. These files will be generated by `npm run build`. 

All changes should be made to the `/src` directory.